docs/_sass/_main.scss

```diff
@@ -445,18 +445,26 @@ button {
     }
 }
 
-.cards-group {
+.cards-group, .platform-cards-group {
     display: grid;
     grid-template-columns: auto;
     row-gap: 20px;
     column-gap: 20px;
     padding-bottom: 20px;
+}
 
+.cards-group {
     @include breakpoint($breakpoint-desktop) {
         grid-template-columns: 50% 50%;
     }
 }
 
+.platform-cards-group {
```

Comment:
Would it be worth extending .cards-group with this one, and just override the breakpoint?

Reply:
I kind of agree that what @dangrous suggests could be a bit cleaner, but I don't think we should treat it as a blocker because it's not a big material difference.

@dangrous if you feel strongly, I'm sure @rushatgabhane would be happy to open a follow-up PR. 🙂 

I'm going to resolve this for now to make sure that our team can start writing all the support articles and iterating on the helpsite in production 🚀

Reply:
yep this is fine to wait!